models/sessionization/segment_web_page_views__sessionized.sql

```diff
@@ -130,7 +130,7 @@ session_ids as (
 
         {{dbt_utils.star(ref('segment_web_page_views'))}},
         page_view_number,
-        {{dbt_utils.surrogate_key('anonymous_id', 'session_number')}} as session_id
```

Comment:
This would require a bump to the dbt-utils version number in the `packages.yml` file, otherwise it will break!

Reply:
Well the inverse is also true, if you bump dbt-utils, this breaks. I wonder if there is a compromise here? I upgraded to 15.2 and it caused this line of code to break.

Reply:
Oh I mean, we should do both. As in, if we use the `[]` syntax, let's make sure the `packages.yml` is updated to reflect that!

Reply:
ohh nice, yea that is a good idea!